pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,61 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  fetchurl
+, lib
+, stdenvNoCC
+, # Dependencies
+  llvmPackages
```

Comment:
Does `clangStdenv` work for building limine instead using `stdenvNoCC` and adding clang/LLVM?

Reply:
For bootloaders you may need to play with `hardeningDisable`, if compiler-hardening related issues pop up.

Reply:
It builds without errors, so when should this be a problem?

Reply:
Some low-level software is picky when certain features are enabled, because it violates their assumptions. Baremetal code is only compatible with `-fstack-protector` when it provides the right symbols, for example.

Reply:
I would suggest using `stdenv` as normal to build the `limine` executable with the default `cc`. This does not affect the compiler used to compile the actual bootloader, which is controlled by `CC_FOR_TARGET` and will default on `clang` if present.

Reply:
done, works